### PR TITLE
Update Maui.Graphics and SkiaSharp

### PR DESCRIPTION
**Pull request by mattleibow**
### Description of Change ###

Update the things!

### CODE REVIEW

File: NuGet.config [hunk 1]
```diff
@@ -3,10 +3,8 @@
   <packageSources>
     <clear />
     <!-- <add key="local" value="artifacts" /> -->
-    <add key="dotnet-eng" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-eng/nuget/v3/index.json" protocolVersion="3" />
-    <add key="dotnet-public" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-public/nuget/v3/index.json" protocolVersion="3" />
```
[the remark this answers is not in the document]
Comment (mattleibow) [reply NuGet.config:7]: ah, yeah

[the remark this answers is not in the document]
Comment (sky41906) [reply NuGet.config:7]: github.com

File: NuGet.config [hunk 2]
```diff
@@ -5,8 +5,8 @@
     <!-- <add key="local" value="artifacts" /> -->
     <add key="dotnet-eng" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-eng/nuget/v3/index.json" protocolVersion="3" />
     <add key="dotnet-public" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-public/nuget/v3/index.json" protocolVersion="3" />
-    <add key="dotnet6" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet6/nuget/v3/index.json" />
-    <!-- <add key="reunion-internal" value="https://pkgs.dev.azure.com/microsoft/ProjectReunion/_packaging/Project.Reunion.nuget.internal/nuget/v3/index.json" /> -->
```
Comment (sky41906) [inline NuGet.config:9]: @sky41906  @github.com

File: NuGet.config [hunk 3]
```diff
@@ -5,8 +5,8 @@
     <!-- <add key="local" value="artifacts" /> -->
     <add key="dotnet-eng" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-eng/nuget/v3/index.json" protocolVersion="3" />
     <add key="dotnet-public" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-public/nuget/v3/index.json" protocolVersion="3" />
-    <add key="dotnet6" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet6/nuget/v3/index.json" />
```
Comment (sky41906) [inline NuGet.config:8]: @sky41906 github.com

**Review by Jun1119jun — COMMENTED** [review]
@sky41906

File: NuGet.config (hunk 1 above)
[the remark this answers is not in the document]
Comment (sky41906) [reply NuGet.config:7]: -  @sky41906 @vs-mobiletools-engineering-service2

**Pull request merged**